.npmrc

```
@@ -0,0 +1,4 @@
+# ignore peerDependencies contract - in preparation of migration to npm 8
+legacy-peer-deps=true
+# reduces noise from console
+loglevel=warn
```

Need a trailing return in this file, otherwise good to go
```suggestion
loglevel=warn

```